docs/pages/Configuration.md

```diff
@@ -540,17 +540,6 @@ This configuration can be overwritten more precisely with the CLI parameters `--
 
 For projects of type `application` and `library` the minification is done for all JavaScript files. While the minfication step debug-variants are created, original resources are minified and source maps are created. It is possible to disable the source map creation.
```

Comment:
This also needs to be updated. It is currently only possible to disable source maps for bundles via the "Custom Bundling" (see section below)